src/components/FilterBar/FilterBar.tsx

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import Button from "./../Button/Button";
+import { ButtonTypes } from "./../Button/ButtonTypes";
+import Heading from "./../Heading/Heading";
+import { HeadingLevels } from "./../Heading/HeadingTypes";
+import Icon from "./../Icons/Icon";
+import { IconRotationTypes, IconSizes } from "./../Icons/IconTypes";
+import Modal from "./../Modal/Modal";
+import { Box, Stack, useMultiStyleConfig } from "@chakra-ui/react";
+import { SelectedItems } from "./../MultiSelect/MultiSelectTypes";
+
+export interface FilterBarProps {
+  /** The id of the FilterBar. */
+  id: string;
+  /** The label of the FilterBar. */
+  label: string;
+  /** Whether the FilterBar modal is open or closed. */
+  isModalOpen: boolean;
+  /** The action to perform on the "filters" button onClick function. */
+  onModalToggle: () => void;
+  /** Whether the FilterBar is in mobile mode or not. */
+  isMobile: boolean;
+  /** The selected items (items that were checked by user). */
+  selectedItems?: SelectedItems;
```

Comment:
@nypl-wluisi With the `MultiSelect` components coming in as children, is the consuming app expected to maintain the state of the `MultiSelect` components in the `Filterbar`?  I see that this prop is used to generate the label text for the mobile "show filters” button.  Is this value also returned in the `onApply` callback?